lib/ibanity/api/base_resource.rb

```diff
@@ -112,5 +116,15 @@ def setup_links(links)
         self[Ibanity::Util.underscore("#{key}_link")] = link
       end
     end
+
+    def relationship_klass(name)
+      camelized_name = Ibanity::Util.camelize(name)
+      enclosing_module = if camelized_name == "FinancialInstitution"
```

Comment:
Why do we have this special case?

Reply:
Because relationships to financial institutions are shared by sandbox resources and XS2A resources, we can't be sure that it's in the same module as the resource.

Reply:
👍